target/linux/ramips/image/mt7621.mk

```diff
@@ -317,6 +317,24 @@ define Device/asus_rt-ax53u
 endef
 TARGET_DEVICES += asus_rt-ax53u
 
+define Device/asus_4g-ax56
+  $(Device/dsa-migration)
+  DEVICE_VENDOR := ASUS
+  DEVICE_MODEL := 4G-AX56
+  IMAGE_SIZE := 51200k
+  UBINIZE_OPTS := -E 5
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  KERNEL_SIZE := 4096k
```

Comment:
Please replace these nand common definitions with `$(Device/nand)`.
ref: https://github.com/openwrt/openwrt/commit/b09a83808a3a4f6ff2ff0cf1f12fdc4193357896

Reply:
Replaced with $(Device/nand) following.  [ref:](https://github.com/openwrt/openwrt/commit/b09a83808a3a4f6ff2ff0cf1f12fdc4193357896)

Reply:
I do not see the changes, did you push them?

Reply:
Apologize,  I will make a new pull request on top of current master. This project was delayed due to other personal stuff.

Reply:
no problem :)
Actually I just acquired a device myself.
Just an offer: I could do the pull request and mention you as the original author in the commit message. I would do the necessary edits.

Unless you want to learn how to do Github and open the new PR yourself. The decision is yours :)

I will support you either way